### ISSUE TITLE
Query from Database Using Plain English

### ISSUE THREAD

**Issue by razakpm** [tech-by-design/polyglot-prime#374]
We need to ensure that Managers and DevOps personnel can query the application database using plain English. This task involves setting up Vanna.ai to facilitate this functionality.

- [ ] Use the existing Infrastructure as Code (IaC) setup to deploy and configure Vanna.ai.
- [ ] Ensure that Vanna.ai is properly connected to the database.
- [ ] Verify that Vanna.ai can interpret and execute queries phrased in plain English.
- [ ] Test with sample queries to ensure the feature is working correctly.
- [ ] Initially, test it in the development environment. After successful completion, promote it to the staging and production environments.

**Comment by ncalarcoTechBD** [comment]
Closing as there is not enough detail or defined business need. Since there is no real direct database integration we don't see a value or need for utilizing Vanna.ai at this time, but we can revisit this if requirements change.
